### ISSUE TITLE
Bug with arguments and Rust macros

### ISSUE THREAD

**Issue by bookcasey** [cursorless-dev/cursorless#905]
Cursorless doesn't recognize arguments passed into Rust macros.

If you try to delete `1` using `chuck arg 1`:

```
fn main() {
    println!("{}", 1);
}
```

It fails with the following error: `Couldn't find containing argumentOrParameter.`

**Comment by auscompgeek** [comment]
I don't think this is trivial to fix. All we get (unsurprisingly) out of tree-sitter is a token tree:
```
> source_file
>> function_item
>>> body: block
>>>> expression_statement
>>>>> macro_invocation
>>>>>> token_tree
>>>>>>> integer_literal
>>>>>>> "1"

**Comment by auscompgeek** [comment]
As a workaround, does the `item` scope do the trick?

**Comment by pokey** [comment]
@auscompgeek good call on "item".  We are actually considering switching "arg" to use "item" under the hood, just restricting allowable parent nodes.  We'd probably do that as follow-up work to #629
